src/Controls/GlobeControls.js

```diff
@@ -552,10 +544,8 @@ class GlobeControls extends THREE.EventDispatcher {
                 dollyEnd.copy(coords);
                 dollyDelta.subVectors(dollyEnd, dollyStart);
 
-                if (dollyDelta.y > 0) {
-                    this.dollyIn();
-                } else if (dollyDelta.y < 0) {
-                    this.dollyOut();
+                if (dollyDelta.y !== 0) {
+                    this.dolly(dollyDelta.y > 0 ? this.dollyInScale : this.dollyOutScale);
```

Comment:
move all tests in `dolly` method?

Reply:
I moved them in `dolly` method, now only the delta value is sent to the method.